en/gps_compass/rtk_gps_drotek_xl.md

```diff
@@ -1,22 +1,8 @@
-# Drotek XL RTK GPS
+# Drotek XL RTK GPS (Discontinued)
```

Comment:
This is no longer available. This is a placeholder - probably delete it next version.